tests/gh78_php7.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Psr\Log\LoggerInterface - Invalid context default value (PHP 7)
+--SKIPIF--
+<?php
+include 'skip.inc';
+if (PHP_VERSION_ID >= 80000) {
+    print "skip Due to version incompatibility";
+}
+?>
+--FILE--
+<?php
+$reflectionMethod = new \ReflectionMethod(\Psr\Log\LoggerInterface::class, "emergency");
+$reflectionParameter = $reflectionMethod->getParameters()[1];
+var_dump($reflectionParameter->isOptional());
+var_dump($reflectionParameter->isDefaultValueAvailable());
+var_dump($reflectionParameter->getDefaultValue());
+--EXPECT--
+bool(true)
+bool(true)
+array(0) {
+}
+--XFAIL--
```

Comment:
Maybe it's better to use EXPECTF here, similar to https://github.com/php/php-src/commit/3709e74b5e495e210ada8039ed81fafa9cbadcdb#diff-2897306bc3a270854edfb5721b5506db844ec9360883a2c07b23efcb021a7db5R24-R25 ?

Reply:
This test is meant as documentation that the behaviour for the extension under PHP 7 isn't perfectly correct, so I think XFAIL is appropriate in this case.